model/dataProvider/providers/ResultsDataProviderService.php

```diff
@@ -24,12 +24,8 @@
 use oat\oatbox\service\ConfigurableService;
 use oat\taoDelivery\model\execution\DeliveryExecution;
 use oat\taoDelivery\model\execution\ServiceProxy;
-use oat\taoDeliveryRdf\helper\DetectTestAndItemIdentifiersHelper;
-use oat\taoResultServer\models\classes\ResultManagement;
-use oat\taoResultServer\models\classes\ResultServerService;
+use oat\taoSync\model\Result\SyncResultDataFormatter;
```

Comment:
Why do we need this data provider outside of taoSyncClient?

Reply:
This `SyncResultDataFormatter` is used in the current synchronization implementation as well as in result file export for synchronization. As I understood the common code will be in `taoSync` client. When eventually the current synchronization will be removed this formatter might be moved to `taoSyncClient`.